### DISCUSSION TITLE
Which browsers are supported?

### DISCUSSION THREAD

**Discussion by nickheal** [vuejs/pinia#1185] [Help and Questions] [answered]
Hi all,

Had my first experience trialling Pinia in a new project at the weekend, and first-off I'm really impressed with it, so congrats to everyone involved on some great work 🎉

Sorry if I'm missing something obvious, but I can't find the intended browser support listed anywhere. Does anyone have a reference for browser support for Pinia 2.0.0?

Many thanks,
Nick

**Comment by posva** [comment, marked as answer]
The same as Vue 3

**Reply by p0358** [reply]
Does that mean that by adding Pinia to a Vue 2 project we are effectively limiting the project's browser support to only what Vue 3 supports? For example dropping support for Internet Explorer?

**Reply by posva** [reply]
No, it aligns with Vue 2.7 in that case.

**Reply by brandonburkett** [reply]
Is this actually correct for vue 2.7? I see errors in edge 15-18 on the pinia import.  Works with edge 79.  I suspect it is object spread operator usage within `pinia.mjs`... which is _not_ used within `pinia.prod.cjs`.

**Reply by posva** [reply]
The spread operator is only used in devtools code which won't be included in production. However, I updated the few occurrences left to use `Object.assign()` in d2a4def

**Reply by brandonburkett** [reply]
Makes sense and thank you @posva 

I am using webpack so devtools were still being included, I posted details on issue [1143](https://github.com/vuejs/pinia/issues/1143) as well as my cjs work around as cjs didn't have the issue at all. .
